### PR TITLE
test(showeditpod): fix for regression FE-2605

### PR DESCRIPTION
**Pull request by KatarzynaQA**
fix test cases which failing during CI regression

### Proposed behaviour

All tests pass

### Current behaviour

Some test cases failing during CI regression

### Checklist

- [ ] Release notes (Conventional Commits) 
- [ ] Unit tests
- [x] Cypress automation tests
- [ ] Storybook added or updated
- [ ] Theme support
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/support/step-definitions/show-edit-pod-steps.js [hunk 1]
```diff
@@ -38,20 +35,18 @@ When('Edit icon has color {string}', (color) => {
...
 });
 
-Then('Show Edit Pod component has border property', () => {
-  showEditPodPreview()
-    .should('not.have.class', `${SHOW_EDIT_POD_BLOCK_CONTENT}no-border`)
-    .and('have.css', 'border', '1px solid rgb(204, 214, 219)');
-  showEditPodPreview().children()
-    .should('not.have.class', SHOW_EDIT_POD_NO_BORDER_CONTENT);
+Then('Show Edit Pod component has border {string} color', (color) => {
+  showEditPodComponent().find('div').eq(0).should('have.css', 'border-bottom-color', color);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/show-edit-pod-steps.js]: magic numbers :)

File: cypress/support/step-definitions/show-edit-pod-steps.js [hunk 2]
```diff
@@ -38,20 +35,18 @@ When('Edit icon has color {string}', (color) => {
...
 });
 
-Then('Show Edit Pod component has no border property', () => {
-  showEditPodPreview()
-    .should('have.class', `${SHOW_EDIT_POD_BLOCK_CONTENT}no-border`)
-    .and('have.css', 'border', '0px none rgba(0, 0, 0, 0.85)');
-  showEditPodPreview().children()
-    .should('have.class', SHOW_EDIT_POD_NO_BORDER_CONTENT);
+Then('Show Edit Pod component has no border {string} color', (color) => {
+  showEditPodComponent().find('div').eq(0).should('have.css', 'border-bottom-color', color);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/show-edit-pod-steps.js]: what is the difference between this TC and the `Show Edit Pod component has border {string} color` ?

File: cypress/support/step-definitions/show-edit-pod-steps.js (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/show-edit-pod-steps.js]: fixed

File: cypress/support/step-definitions/show-edit-pod-steps.js (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/show-edit-pod-steps.js]: fixed

File: cypress/support/step-definitions/show-edit-pod-steps.js [hunk 3]
```diff
@@ -38,20 +35,11 @@ When('Edit icon has color {string}', (color) => {
...
-  showEditPodPreview()
-    .should('have.class', `${SHOW_EDIT_POD_BLOCK_CONTENT}no-border`)
-    .and('have.css', 'border', '0px none rgba(0, 0, 0, 0.85)');
-  showEditPodPreview().children()
-    .should('have.class', SHOW_EDIT_POD_NO_BORDER_CONTENT);
+Then('Show Edit Pod component has border {string} color', (color) => {
+  showEditPodComponent().find('div').eq(FIRST_DIV).should('have.css', 'border-bottom-color', color);
+  showEditPodComponent().find('div').eq(FIRST_DIV).should('have.css', 'border-left-color', color);
+  showEditPodComponent().find('div').eq(FIRST_DIV).should('have.css', 'border-right-color', color);
+  showEditPodComponent().find('div').eq(FIRST_DIV).should('have.css', 'border-top-color', color);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/show-edit-pod-steps.js]: could we use `.and` like chain?

File: cypress/support/step-definitions/show-edit-pod-steps.js [hunk 4]
```diff
@@ -77,17 +64,15 @@ Then('Show Edit Pod component hasn\'t a cancel button', () => {
 Then('Show Edit Pod classic component has saving property', () => {
   showEditPodSaveButton().should('be.disabled');
   showEditPodSaveButton().should('have.attr', 'disabled');
-  showEditPodSaveButton()
-    .should('have.css', 'background', 'rgb(230, 235, 237) none repeat scroll 0% 0% / auto padding-box border-box')
+  showEditPodSaveButton().should('have.css', 'background', 'rgb(230, 235, 237) none repeat scroll 0% 0% / auto padding-box border-box')
```
Comment (DlgSHi) [inline cypress/support/step-definitions/show-edit-pod-steps.js]: the same as above, `.and`

File: cypress/support/step-definitions/show-edit-pod-steps.js [hunk 5]
```diff
@@ -77,17 +64,15 @@ Then('Show Edit Pod component hasn\'t a cancel button', () => {
...
     .and('have.css', 'border', '1px solid rgba(0, 0, 0, 0)')
     .and('have.css', 'color', 'rgba(0, 0, 0, 0.2)');
 });
 
 Then('Show Edit Pod component has saving property', () => {
   showEditPodSaveButton().should('be.disabled');
   showEditPodSaveButton().should('have.attr', 'disabled');
-  showEditPodSaveButton()
-    .should('have.css', 'background', 'rgb(229, 234, 236) none repeat scroll 0% 0% / auto padding-box border-box')
+  showEditPodSaveButton().should('have.css', 'background', 'rgb(229, 234, 236) none repeat scroll 0% 0% / auto padding-box border-box')
```
Comment (DlgSHi) [inline cypress/support/step-definitions/show-edit-pod-steps.js]: the same as above, `.and`

Comment (KatarzynaQA) [reply cypress/support/step-definitions/show-edit-pod-steps.js]: fixed

File: cypress/support/step-definitions/show-edit-pod-steps.js (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/show-edit-pod-steps.js]: fixed

File: cypress/support/step-definitions/show-edit-pod-steps.js (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/show-edit-pod-steps.js]: fixed

**Pull request merged**